UnityMixedCallstackFilter.cs

```diff
@@ -11,19 +11,18 @@
 
 namespace UnityMixedCallstack
 {
-    public class UnityMixedCallstackFilter : IDkmCallStackFilter, IDkmLoadCompleteNotification
+    public class UnityMixedCallstackFilter : IDkmCallStackFilter, IDkmLoadCompleteNotification, IDkmModuleInstanceLoadNotification
     {
         private static List<Range> _rangesSortedByIp = new List<Range>();
         private static FuzzyRangeComparer _comparer = new FuzzyRangeComparer();
-        private static bool _enabled = true;
+        private static bool _enabled;
         private static IVsOutputWindowPane _debugPane;
         private static string _currentFile;
         private static FileStream _fileStream;
         private static StreamReader _fileStreamReader;
 
         public void OnLoadComplete(DkmProcess process, DkmWorkList workList, DkmEventDescriptor eventDescriptor)
```

Comment:
Future Cleanup: We could probably remove this window stuff, unless we're planning on logging things to the output window.

Reply:
I thought about it, but I see we log other info there as well.